configs/glean/glean_in128out1024_300k-4xb2_ffhq-celebahq.py

```diff
@@ -1,7 +1,8 @@
 _base_ = '../_base_/models/base_glean.py'
 
-experiment_name = 'glean_in128out1024_4x2_300k_ffhq_celebahq'
+experiment_name = 'glean_in128out1024_300k-4xb2_ffhq-celebahq'
```

Comment:
`ffhq-celeba-HxW`

Reply:
in128out1024 refer to the structure of networks, and we only list dataset here, resolution is not necessary